### ISSUE TITLE
[Feature Request] Download resoures on the go

### ISSUE THREAD

**Issue by rampr** [gameclosure/devkit#231]
Instead of the build having all the static resources(images, fonts and audio), they could be downloaded on the go. Helps in 2 cases(there could be more use cases)
1. Reduces the build size
2. Avoid releasing new updates just for content updates, can be managed by the game itself.

**Comment by mgh** [comment]
Definitely! This is on our TODO list.  Some preliminary work has to be done first before we can support this (I think @jwilm is starting on this).

**Comment by rampr** [comment]
Awesome :)
Can you throw more light on the preliminary work ?

**Comment by jwilm** [comment]
@rampr we're adding native support for downloading entire applications. Support for that should be showing up in early summer. DLC packs like you're asking for are further down the priority list and would be September/October at the earliest.
